### PR TITLE
Add dg/bypass-finals to the conflict packages list

### PR DESCRIPTION
**Pull request by alex-ception**
This PR adds the library dg/bypass-finals to the conflict section of composer.json file since infection and bypass-finals aren't compatibles yet.

Related to https://github.com/infection/infection/issues/1275

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -67,7 +67,8 @@
         "webmozart/path-util": "^2.3"
     },
     "conflict": {
-        "phpunit/php-code-coverage": ">9 <9.1.4"
+        "phpunit/php-code-coverage": ">9 <9.1.4",
+	    "dg/bypass-finals": "*"
```
Comment (maks-rafalko) [inline composer.json:71]:
```suggestion
	      "dg/bypass-finals": "*"
```

**Review by maks-rafalko — APPROVED** [review]

**Review by sanmai — APPROVED** [review]

**Comment by sanmai** [comment]
I feel like this is a conflict for `infection/include-interceptor` not for Infection itself.

See https://github.com/infection/include-interceptor/pull/13 https://github.com/dg/bypass-finals/issues/9

File: composer.json (hunk 1 above)
Comment (maks-rafalko) [reply composer.json:71]: also, please run `composer update --lock` to fix

> - The lock file is not up to date with the latest changes in composer.json, it is recommended that you run `composer update` or `composer update <package name>`.

**Comment by maks-rafalko** [comment]
> I feel like this is a conflict for `infection/include-interceptor` not for Infection itself.

yes and no. Someone should still be able to use interceptor (for whatever reason) without Infection, even if they use bypass-finals.

So let's add it here for now just to avoid Infection's users time waste. I really hope we will soon revert it if bypass-finals's author answers to me here https://github.com/dg/bypass-finals/issues/9#issuecomment-980070046

**Pull request merged**